### PR TITLE
[DOCS] Update fluent In Progress Cautionary Note

### PR DESCRIPTION
**Pull request by kwcanuck**
Update to in progress message

### CODE REVIEW

**Comment by kwcanuck** [comment]
@don thanks for the edit. As discussed, the meaning of _incompatibilities_ is unclear. Also, our style guide recommends that we avoid first-person pronouns (our) unless we refer to the organization first. For example: _Great Expectations recommends that you feed your aardvark Standardized Aardvark Treats. We cannot guarantee the happiness of your aardvark otherwise._ Here is what I propose:

_Great Expectations is aware that the fluent datasource and yaml config workflows are undocumented. A new method for connecting to data sources will be added to this document soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance._

File: docs/docusaurus/docs/components/warnings/_in_progress.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 :::caution In progress
 
-The fluent datasources and yaml configuration are not working as expected. A new method for connecting to data sources will be added to this document soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
+There are currently broken workflows and incompatibilities with our fluent datasources and yaml config. This document is under review and a new method for connecting to data sources will be added soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/components/warnings/_in_progress.md]:
```suggestion
Great Expectations is aware that the fluent datasource and yaml config workflows are undocumented. A new method for connecting to data sources will be added to this document soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
```

**Comment by kwcanuck** [comment]
Feedback from @donaldheppner on the revision proposed in the previous comment:

image

File: docs/docusaurus/docs/components/warnings/_in_progress.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 :::caution In progress
 
-The fluent datasources and yaml configuration are not working as expected. A new method for connecting to data sources will be added to this document soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
+Great Expectations is aware that the fluent datasource and yaml config workflows are undocumented. A new method for connecting to data sources will be added to this topic soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
```
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/components/warnings/_in_progress.md:3]:
```suggestion
Great Expectations is aware that the fluent Datasource and yaml config workflows are undocumented. A new method for connecting to a source data system will be added to this topic soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
```

"Datasource" all one word is the technical term, so our convention has been to capitalize it.  To lesson confusion with "data sources" we've been using "source data systems" to refer to data as it exists outside of GX.

**Review by Rachel-Reverie — APPROVED** [review]

**Pull request merged**